src/styles/tailwind.css

```
@@ -6,8 +6,9 @@
 }
 
 @theme static {
-    --font-serif: 'Times New Roman', serif;
-    --font-sans: 'Arial', sans-serif;
+    --font-sans: var(--custom-font-sans);
+    /* --font-serif: var(--custom-font-serif); */
+    /* --font-mono: var(--custom-font-mono); */
```

Removed unused commented variables for clarity
```suggestion
    --font-sans: var(--custom-font-sans);
```